setup/src/common/scripts/postStartCommand.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# Copyright (c) 2023 Robert Bosch GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+echo "#######################################################"
+echo "### Auto-Upgrade CLI                                ###"
+echo "#######################################################"
+
+# Check for internet connection
+wget -q --spider http://google.com
```

Comment:
Not necessary to do an additional web request, better to simply exit early if line 38 does fail